src/v2/Apps/Auction/Components/Form/Utils/errorMessages.ts

```diff
@@ -35,6 +36,8 @@ const BIDDING_STATE_TO_MESSAGE: Record<
   RESERVE_NOT_MET:
     "Your bid is below the reserve price. Please select a higher bid.",
   SALE_CLOSED: "This sale had been closed. Please browse other open sales.",
```

Comment:
unrelated but 'had been closed' somehow sounds bizarre to me, vs. what you put for `LOT_CLOSED` ('has closed')

Reply:
Terrible grammar. I'll update the sale one while i'm here.